app/controllers/admin/merchants_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Admin::MerchantsController < ApplicationController
```

Comment:
This is a controller that uses a module a la name spacing. There are still some aspects of this I am struggling with but I'm pretty sure we need to implement it across the board.